### ISSUE TITLE
Cree search with yâhyânam gives two, duplicate results

### ISSUE THREAD

**Issue by aarppe** [UAlbertaALTLab/cree-intelligent-dictionary#372]
When searching with _yâhyânam_, we get two results which are duplicates of each other (and a duplicate of _nihtâ-yâhyânam_ as well, cf.

![image](https://user-images.githubusercontent.com/15003407/77808561-7a048580-7051-11ea-967a-dd68963abdda.png)

Of course, we should see the same dictionary entries for _yâhyânam_ and  _nihtâ-yâhyânam_ only once each.

**Comment by aarppe** [comment]
@Madoshakalaka One more discrepancy for you, all having to do with _swimming_.

**Comment by aarppe** [comment]
The two analyses seem to arise from the verb having two matches in CW, as an `AI` and `TI` one. The English translations convey both an AI and an TI reading - and it might be that this is a special case of VAI's that can sometimes take a object (VAI-O's) which are apparent in
other related languages such as East Cree. However, for Plains Cree, Arok has not designated VAI-O as a distinct word-class (for now).

However, the current dictionary source I have has only one dictionary entry:

```
egrep '^ýâhýânam' ~/GDrive/CreeFST/Wolvengrey/Wolvengrey_crk2eng_200202_altlab.tsv | cut -f1,3,4
ýâhýânam	VAI-3	s/he swims; s/he swims in s.t.
```

However, the current XML source does have two dictionary entries for the verb in question.

```
grep -a3 '>yâhyânam' ~/giella/art/dicts/crk/crkeng_cw_md_200314.xml

<e>
   <lg>
      <l pos="V">yâhyânam</l>
      <lc>VTI-1</lc>
      <stem>yâhyân-</stem>
   </lg>
--
--

<e>
   <lg>
      <l pos="V">yâhyânam</l>
      <lc>VAI-3</lc>
      <stem>yâhyân-</stem>
   </lg>
```

Reconstructing what has happened it seems that Arok had coded _yâhyânam_ as a `VTI` years ago (since it morphologically takes `VTI` suffixes), and then decided that it was rather a `VAI` (removing the `VTI` reading). When I incorporated Arok's newest database, I extended the existing `crkeng*.xml `rather than creating it from scratch, and that left any such preceding dictionary entries intact.

So, it seems that this is a dictionary source XML issue, not a software one. We do want to be able to generate the dictionary source (without XML) directly from the dictionary DB now in the works.

**Comment by aarppe** [comment]
@Madoshakalaka so it seems that this is primarily for me to fix, if we want to fix it, but removing the VTI dictionary entry for _yâhyânam_ in the XML source.

**Comment by aarppe** [comment]
Now you get a slightly different version of this, but incorrect behavior nonetheless:

<img width="363" alt="image" src="https://user-images.githubusercontent.com/15003407/83337312-7242a680-a277-11ea-8488-61b7344d6a06.png">

**Comment by aarppe** [comment]
This duplicate case with _yâhyânam_ and a number of others have now been removed from crkeng.xml, as part of #420, but some cases require consultation with AEW, just to be sure. In this respect, this particular issue is a sub-case of #420.
